src/sentry/api/endpoints/project_releases.py

```diff
@@ -78,12 +83,10 @@ def get(self, request, project):
                 projects=project, organization_id=project.organization_id
             ).select_related('owner')
             if environment is not None:
-                # TODO(LB): May want to change this to ReleaseProjectEnv don't see a
-                # reason to change now.
-                queryset = queryset.filter(id__in=ReleaseEnvironment.objects.filter(
-                    organization_id=project.organization_id,
-                    environment_id=environment.id,
-                ).values_list('release_id', flat=True))
+                queryset = queryset.filter(
+                    releaseprojectenvironment__project=project,
+                    releaseprojectenvironment__environment=environment,
+                )
```

Comment:
Needs distinct or maybe switch back to using `IN`

Reply:
Actually it's fine, because we're filtering on all the unique attributes

Reply:
:+1: for using a join.